source/fe/fe_tools.cc

```diff
@@ -785,7 +785,7 @@ namespace FETools
       tria.begin_active()->set_refine_flag (RefinementCase<dim>(ref_case));
       tria.execute_coarsening_and_refinement ();
 
-      MappingQ1<dim,spacedim> mapping;
+      MappingQGeneric<dim,spacedim> mapping(1);
```

Comment:
Why not delete the variable `mapping` here altogether? It is only used in FEValues which can be initialized without a mapping, taking the value of `mapping`.